Aptoide/aptoide-client-v8
# [ASV-1349] Created WizardManager to handle the creation of the wizard for cobrand.

**What does this PR do?**

   This PR aims to remove the second page of the wizard for the cobrand flavor.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] WizardFragment.java

**How should this be manually tested?**

  Install both the vanilla and cobrand flavors and check if the wizard have the correct amount of pages (2 on cobrand, and 3 on vanilla).

**What are the relevant tickets?**

  https://aptoide.atlassian.net/browse/ASV-1349




**Code Review Checklist**
